src/query/storage/cache/redis_cache.go

```diff
@@ -372,6 +425,10 @@ func BucketWindowGetOrFetch(
 			res, err := st.FetchProm(ctx, expanded_query, fetchOptions)
 			cache.cacheMetrics.CacheMetricsMiss(res)
 			if err == nil {
+				// Don't set last bucket if it's within a minute of the end of query to ensure M3DB is filled out when we set
```

Comment:
put more comments why we should not set within a minute.

Reply:
added